charts/cinder-csi-plugin/values.yaml

```diff
@@ -160,6 +160,12 @@ secret:
 
 storageClass:
   enabled: true
+  parameters:
+    enabled: false
+    type: gp2
+    availabilityZone: us-east-1a
```

Comment:
I am talking about here, e.g apparently we don't want any chart default to this AWS region?

Reply:
I got your point, let me fix this real quick